addons/website/views/snippets/s_alert.xml

```diff
@@ -12,19 +12,24 @@
 
 <template id="s_alert_options" inherit_id="website.snippet_options">
     <xpath expr="." position="inside">
-        <div data-selector=".s_alert">
+        <div data-selector=".s_alert" data-js="Alert">
             <we-colorpicker string="Color"
                 data-select-style="true"
                 data-css-property="background-color"
                 data-color-prefix="alert-"/>
-        </div>
-
-        <div data-selector=".s_alert">
             <we-select string="Size">
                 <we-button data-select-class="s_alert_sm">Small</we-button>
                 <we-button data-select-class="s_alert_md">Medium</we-button>
                 <we-button data-select-class="s_alert_lg">Large</we-button>
             </we-select>
+            <we-select string="Type" data-apply-to=".fa">
+                <we-button data-select-class="fa-user-circle" data-select-type="alert-primary">Primary</we-button>
```

Comment:
Reading this, I think that if someone puts icons inside an alert, changing the alert type will change all the icons he put. Probably need a specific class on the icon to change ?

Reply:
Good catch, done